k8s-bench/tasks/fix-probes/setup.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Delete namespace if exists and create a fresh one
+kubectl delete namespace health-check --ignore-not-found
+kubectl create namespace health-check
+
+# Create a deployment with problematic health checks
+cat <<YAML | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webapp
+  namespace: health-check
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webapp
+  template:
+    metadata:
+      labels:
+        app: webapp
+    spec:
+      containers:
+      - name: webapp
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+        # The problem: incorrect health probes causing restarts
+        livenessProbe:
+          httpGet:
+            path: /nonexistent-path  # Path doesn't exist
```

Comment:
I have made this mistake in the past with other evals.

I think the path sort of contains hint for the AI models. Lets make it harder for the AI.

Some suggestions for mis-configured paths: "/is_live", "/get_status" or "/get_health"

similarly for the readiness below.